src/pages/groups/Group.js

```diff
@@ -103,18 +103,20 @@ const GroupDetails = ({ groupName }) => {
         h(NewUserCard, {
           onClick: () => setCreatingNewUser(true)
         }),
-        h(MemberCardHeaders, { sort, onSort: setSort }),
-        div({ style: { flexGrow: 1, marginTop: '1rem' } },
-          _.map(member => {
-            return h(MemberCard, {
-              adminLabel: 'admin',
-              userLabel: 'member',
-              member, adminCanEdit,
-              onEdit: () => setEditingUser(member),
-              onDelete: () => setDeletingUser(member)
-            })
-          }, _.orderBy([sort.field], [sort.direction], _.filter(({ email }) => Utils.textMatch(filter, email), members)))
-        ),
+        div({ role: 'table', 'aria-label': 'member list' }, [
```

Comment:
This looks very similar to https://github.com/DataBiosphere/terra-ui/pull/2531/files#diff-63f09855608a6a241dc6a5b78476307d1ea6ef749c662b0bf3323c2fc87a8f28R101, do you think we could extract to a component? (I was hesitant before, because there were different stylings, but if we are wrapping in identical divs, that feels componentable.)

Reply:
Yep, I was thinking the same thing, and was curious why there were different stylings. As in, was that intentional, or should they just be the same?

At any rate, I won't make any further changes to this PR right now, since #2532 supercedes portions of it.